include/frameAndLink.h

```diff
@@ -0,0 +1,92 @@
+#ifndef FRAMEANDLINK_H
+#define FRAMEANDLINK_H
+
+#include "typedefs.h"
+#include "transformationBuffer.h"
+#include "stampedTransformation.h"
+
+#include <string>
+#include <vector>
+#include <QtGui/QQuaternion>
+
+class Frame;
+
+/***************************
+ * LINK                    *
+ ***************************/
+
+class Link
+{
+
+friend class GMLWriter;
+
+public:
+
+    Link(Frame *const _parent, Frame *const _child, const DurationSec &_storageTime)
+    : parent{_parent}
+    , child(_child)
+    , buf(_storageTime)
+    {}
+
+    bool oldestTransformation(const FrameID &srcFrame, StampedTransformation &stampedTransformation);
```

Comment:
This method should be `const` if it does not modify the `Link` object. The same comment applies to the next ones.